test/e2e/set/scyllacluster/scyllacluster_replace.go

```diff
@@ -0,0 +1,88 @@
+// Copyright (C) 2021 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+	"github.com/scylladb/scylla-operator/pkg/naming"
+	scyllaclusterfixture "github.com/scylladb/scylla-operator/test/e2e/fixture/scyllacluster"
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = g.Describe("ScyllaCluster replace", func() {
+	defer g.GinkgoRecover()
+
+	f := framework.NewFramework("scyllacluster")
+
+	g.It("should replace a non-seed node", func() {
+		ctx, cancel := context.WithTimeout(context.Background(), testTimout)
+		defer cancel()
+
+		scyllaCluster := scyllaclusterfixture.BasicScyllaCluster.ReadOrFail()
+		scyllaCluster.Spec.Datacenter.Racks[0].Members = 3
+
+		framework.By("Creating a ScyllaCluster")
+		err := framework.SetupScyllaClusterSA(ctx, f.KubeClient().CoreV1(), f.KubeClient().RbacV1(), f.Namespace(), scyllaCluster.Name)
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		scyllaCluster, err = f.ScyllaClient().ScyllaV1().ScyllaClusters(f.Namespace()).Create(ctx, scyllaCluster, metav1.CreateOptions{})
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		framework.By("Waiting for the ScyllaCluster to deploy")
+		waitCtx1, waitCtx1Cancel := contextForRollout(ctx, scyllaCluster)
+		defer waitCtx1Cancel()
+		scyllaCluster, err = waitForScyllaClusterState(waitCtx1, f.ScyllaClient().ScyllaV1(), scyllaCluster.Namespace, scyllaCluster.Name, scyllaClusterRolledOut)
```

Comment:
some lines are too long

Reply:
yeah, but it least to me it kind of still looks better then every simple arg on it's line:
```
scyllaCluster, err = waitForScyllaClusterState(
    waitCtx1,
    f.ScyllaClient().ScyllaV1(),
    scyllaCluster.Namespace,
    scyllaCluster.Name,
    scyllaClusterRolledOut,
)
```
90s+ editors are pretty good with wrapping lines
e.g. Kubernetes e2e uses the same https://github.com/kubernetes/kubernetes/blob/303fd4aca2ced102f2698beaadea4f1e1417ec5b/test/e2e/apps/deployment.go#L351